corporate/urls.py

```diff
@@ -27,13 +27,13 @@
     hello_view,
     landing_view,
     plans_view,
+    remote_realm_plans_page,
+    remote_server_plans_page,
```

Comment:
It would probably be better to just extract a new `corporate.views.plans` module.

Reply:
I am just add a commit later for this.